### PR TITLE
Update project for release 0.3.0

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

- Update workflow.yml file
- Update GH repo links: the owner was still `marcofavorito` instead of the new owner `whitemech`.
- Add more clean commands in `Makefile`
- Make `Pipfile` be only for development environment
- Update badges in README
- Increase `pythomata` version number lowerbound.

## Fixes
N/A

## Types of changes

What types of changes does your code introduce to agents-tac?
_Put an `x` in the boxes that apply_

- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [X] I have read the [CONTRIBUTING](../master/CONTRIBUTING.rst) doc
- [X] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [X] Lint and unit tests pass locally with my changes
- [X] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by cipollone** [comment]
Thanks for these corrections.

Why did you remove the direct dependencies from Pipfile? Maybe we could move the to a new `[packages]` section.

**Comment by marcofavorito** [comment]
I think we could use `Pipfile` only for the development environment. The dependencies I removed can be installed as part of the package: `pip install .`.

That is linked to the discussion [Pipfile vs setup.py](https://pipenv-fork.readthedocs.io/en/latest/advanced.html#pipfile-vs-setuppy). In fact, `Pipfile` is more useful for an application or development environment rather than for managing dependencies of a library, specified in `install_requires` in `setup.py`.

But happy to discuss, maybe it's not a good idea, or it can be improved.

File: HISTORY.md [hunk 1]
```diff
@@ -1,34 +1,45 @@
 # History
 
-## 0.1.0 (2018-04-11)
+## 0.3.0 (2020-03-XX)
```
Comment (marcofavorito) [inline HISTORY.md]: we will update the day as soon as we are ready

**Comment by cipollone** [comment]
> That is linked to the discussion [Pipfile vs setup.py](https://pipenv-fork.readthedocs.io/en/latest/advanced.html#pipfile-vs-setuppy).
Since, we don't have Pipfile.lock in our repository, there should be no disadvantage in keeping them in Pipfile, i guess.
However, I agree with you, because this way we don't have duplicated dependencies to maintain.

File: AUTHORS.md [hunk 2]
```diff
@@ -8,4 +8,8 @@
 
 * [Roberto Cipollone](https://github.com/cipollone) <[cipollone.rt@gmail.com](mailto:cipollone.rt@gmail.com)>
 
-[How to contribute](./CONTRIBUTING.md)? 
+
+## Contributors
+
```
Comment (cipollone) [inline AUTHORS.md]: Duplicated section.

**Comment by cipollone** [comment]
Apart from the failed tests (which I don't know), LGTM.

**Comment by marcofavorito** [comment]
> tests

they are due to coverage, but let's don't care atm.

File: AUTHORS.md (hunk 2 above)
Comment (marcofavorito) [reply AUTHORS.md]: oh sorry...

**Pull request merged**